Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -795,6 +795,18 @@ def audit_specs
         return if stable_url_minor_version.even?
 
         problem "#{stable.version} is a development release"
+      when %r{^https://github.com/}
+        matches = stable.url.match(%r{https://github.com/(.*)/.*/(releases/download/(.*)/.*tar.gz|archive/(.*).tar.gz)})
```

Comment:
Matching `*tar.gz` for URLs with `/releases/download/` probably won't work since we support a lot of different formats (`.tar.xz`, `.tar.bz2`, `.zip`, `.jar`, `.phar`, etc).

I'd go with several regexes which make it much easier to read